docs/source/en/pipeline_tutorial.mdx

```diff
@@ -39,7 +39,9 @@ While each task has an associated [`pipeline`], it is simpler to use the general
 2. Pass your input text to the [`pipeline`]:
 
 ```py
->>> generator("Three Rings for the Elven-kings under the sky, Seven for the Dwarf-lords in their halls of stone")
+>>> generator(
+...     "Three Rings for the Elven-kings under the sky, Seven for the Dwarf-lords in their halls of stone"
+>>> )  # doctest: +SKIP
```

Comment:
Thos `>>>` will be fixed very soon, if we can merge after the fix is out and `doc-builder` gets a new release, that would be great!